src/main/scala/edg_ide/runner/CompileProcessHandler.scala

```diff
@@ -297,10 +297,7 @@ class CompileProcessHandler(project: Project, options: DesignTopRunConfiguration
           indicator.setText("EDG compiling: generating PDF")
           console.print("Printing PDF\n", ConsoleViewContentType.LOG_INFO_OUTPUT)
           val (pdfGeneration, pdfTime) = timeExec {
-            PDFGeneratorUtil.generate(
-            HierarchyGraphElk.HBlockToElkNode(compiled.getContents,
-              mappers=Seq(new ElkEdgirGraphUtils.TitleMapper(compiler))),
-            options.pdfFile)
+            PDFGeneratorUtil.generate(compiled.getContents, mappers=Seq(new ElkEdgirGraphUtils.TitleMapper(compiler)),options.pdfFile)
```

Comment:
@ducky64 is this the right way for me to use the new refdes name mapping feature?

Reply:
Yep! Looks like it works too!